components/FormikFields/PaperInputPicker/index.js

```diff
@@ -234,6 +234,7 @@ const PaperInputPicker = ({
             <View key={result.value}>
               {result.text === true && result.value === values[formikKey] && (
                 <View style={stylesDefault} key={result.textKey}>
+                  <Text>{customForm ? result.textQuestion : I18n.t(result.textQuestion)}</Text>
```

Comment:
@jamccomb92 not all custom forms are going to have this though right?

if `customForm` is true but there's no `textQuestion` this will break